app/(voters)/candidate/[slug]/ElectionTab.tsx

```
@@ -1,107 +1,142 @@
+                <div
+                  key={block.id}
+                  className={`text-sm ${color}`}
+                  dangerouslySetInnerHTML={{
+                    __html: mdToHtml(block.body ?? ""),
+                  }}
+                />
+              );
+
+            case BlockType.LIST:
+              const ListTag =
+                block.listStyle === ListStyle.NUMBER ? "ol" : "ul";
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)